ext/CatalystGraphMakieExtension.jl

```diff
@@ -1,11 +1,13 @@
 module CatalystGraphMakieExtension
 
 # Fetch packages.
-using Catalyst, GraphMakie
+using Catalyst, GraphMakie, Graphs 
 import Catalyst: lattice_plot, lattice_animation, extract_vals
-import Graphs: AbstractGraph, SimpleGraph
+import Graphs: AbstractGraph, SimpleGraph, SimpleDiGraph, SimpleEdge, src, dst, ne, nv 
+import Catalyst: speciesreactiongraph, incidencematgraph
 
 # Creates and exports hc_steady_states function.
 include("CatalystGraphMakieExtension/graph_makie_extension_spatial_modelling.jl")
-
+include("CatalystGraphMakieExtension/rn_graph_plot.jl")
+export SRGraph, ComplexGraph
```

Comment:
In Catalyst.jl have

```julia
function plot_speciesreaction_graph end
function plot_complex_graph end
export plot_speciesreaction_graph, plot_complex_graph
```
Note you also need to update the doc strings below for the new names.

Reply:
Actually, for consistency it should probably be
```julia
plot_species_reaction_graph
```

Reply:
Which admittedly is very long. Can we think of a shorter descriptive name?

Reply:
plot_sr_graph?

Reply:
What about just `plot_network(...)` and `plot_complexes(...)`?